tests/output.rs

```diff
@@ -1113,3 +1120,67 @@ fn test_only_matching_multiline_overlapping_matches_starting_on_same_line() {
         "#,
     );
 }
+
+#[test]
+fn test_byte_offset() {
+    assert_sorted_output(
+        "rust_project",
+        r#"
+            $ tree-sitter-grep -q '(function_item) @function_item' -l rust --byte-offset
+            src/helpers.rs:1:0:pub fn helper() {}
+            src/stop.rs:1:0:fn stop_it() {}
+            src/lib.rs:3:14:pub fn add(left: usize, right: usize) -> usize {
+            src/lib.rs:4:63:    left + right
+            src/lib.rs:5:80:}
+            src/lib.rs:12:139:    fn it_works() {
+            src/lib.rs:13:159:        let result = add(2, 2);
+            src/lib.rs:14:191:        assert_eq!(result, 4);
+            src/lib.rs:15:222:    }
```

Comment:
So the second number here is the offset? I guess I've seen this before, but assumed without looking to hard it was the column. (Or is there another identically formatted grep output style that does work like that?)

Reply:
Ya it is kind of weird that it's the same "format" as eg `--vimgrep` mode where you have line number + column

So ya here the second # is the absolute byte offset

In the `test_byte_offset_vimgrep()` test below you can see that it is possible to combine those and get `row:column:byte_offset`